.github/workflows/reusable-ubuntu.yml

```diff
@@ -73,14 +73,27 @@ jobs:
         --enable-slower-safety
         --with-openssl=$OPENSSL_DIR
         ${{ fromJSON(inputs.free-threading) && '--disable-gil' || '' }}
+    - name: Build CPython out-of-tree (for compiler warning check)
+      if: ${{ !inputs.free-threading}}
+      working-directory: ${{ env.CPYTHON_BUILDDIR }}
+      run: set -o pipefail; make -j4 2>&1 | tee compiler_output_ubuntu.txt
```

Comment:
Nit: same thing here: let's move this step after the "Build CPython out-of-tree" step.

Reply:
Rearranged step order